ftp/src/main/scala/akka/stream/alpakka/ftp/javadsl/FtpApi.scala

```diff
@@ -230,6 +228,28 @@ sealed trait FtpApi[FtpClient] { _: FtpSourceFactory[FtpClient] =>
       .asJava
   }
 
+  /**
+   * Java API for creating a directory in a given path
+   *
+   * @param basePath path to start with
+   * @param name name of a directory to create
+   * @param connectionSettings connection settings
+   * @return [[akka.stream.javadsl.Source Source]] of [[Done]]
+   */
+  def mkdirSource(basePath: String, name: String, connectionSettings: S): Source[Done, NotUsed] =
```

Comment:
Hmm, makes sense but i wanted to make it similar to S3 API, which is should be the same since it is the same library:
https://github.com/akka/alpakka/blob/7dc6727bd2e02e562f0097f316017f2754211a0e/s3/src/main/scala/akka/stream/alpakka/s3/scaladsl/S3.scala#L248

Reply:
I think keeping consistency across a single module is more important. Unifying the naming across multiple modules could be done in a separate PR if deemed worth it.

Reply:
Ok, changed